provider/pkg/arm2pulumi/internal/test/s3-upload-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -euf -o pipefail
```
Since you are using pipes to make sure we catch failures. (Test and make sure this works first).